vagrant/maestro.config

```diff
@@ -51,6 +51,9 @@ targets:
       - levels: error
         format_pre: "\u001B[31m"    # red
         format_post: "\u001B[39m"
+      - levels: info
```

Comment:
Do we want to keep this in or can it be removed?

Reply:
I'd like to keep it especially since it's just the vagrant's maestro.config because I spent some time debugging trying to figure out why my log messages weren't being printed.